include/seqan3/core/tuple_utility.hpp

```diff
@@ -199,3 +199,26 @@ constexpr auto tuple_pop_front(tuple_t && t)
 }
 //!\}
 } // namespace seqan3
+
+namespace seqan3::detail
+{
+
+//!\brief Value metafunction that returns whether a type is an element type of a tuple. [Base template]
+template <typename ...t>
+inline bool constexpr is_element_of = false;
+
+//!\brief Value metafunction that returns whether a type is an element type of a tuple. [Overload to prevent recursion]
+template <tuple_like_concept tuple_t>
+inline bool constexpr is_element_of<tuple_t, tuple_t> = false;
+
+//!\brief Value metafunction that returns whether a type is an element type of a tuple. [Auxiliary overload]
+template <typename element_t, tuple_like_concept tuple_t, size_t ...idx>
+inline bool constexpr is_element_of<element_t, tuple_t, std::index_sequence<idx...>> =
+    (std::Same<element_t, std::tuple_element_t<idx, tuple_t>> || ...);
```

Comment:
to suppo
```c++
std::Same<element_t, std::tuple_element_t<idx, tuple_t>> || ...
```
to
```c++
std::Same<element_t, std::tuple_element_t<idx, tuple_t>> || ... || true
```

Reply:
?

Reply:
My comment was truncated: to support empty tuples